taqueria-plugin-ligo/compile.ts

```diff
@@ -1,17 +1,21 @@
-const {execCmd, getArch, sendAsyncErr, sendJsonRes, sendErr} = require('@taqueria/node-sdk')
-const {extname, basename, join} = require('path')
-const glob = require('fast-glob')
+import {execCmd, getArch, sendAsyncErr, sendJsonRes, sendErr} from '@taqueria/node-sdk'
+import {SanitizedArgs, PluginResponse, Failure, LikeAPromise} from "@taqueria/node-sdk/types";
+import {extname, basename, join} from 'path'
+// @ts-ignore interop issue. Maybe find a different library later
```

Comment:
@jchenche what is the ts-ignore for?

Reply:
Without it, it will give me the error: `can only be default-imported using the 'esModuleInterop' flag`

We've seen this before and solved it using `@ts-ignore` due to some 3rd-party library type issue, if I recall correctly.

Reply:
Mind trying if `import * as glob from 'fast-glob'` works please?

Reply:
Hmm it works though I don't know why the editor is complaining (same version of TS). Can we ignore the complain for VS Code?

![Screen Shot 2022-05-02 at 15 14 15](https://user-images.githubusercontent.com/30018387/166335770-bb7792e1-b4e0-485b-84cb-66fc0da7f1a2.png)

Reply:
However, `import glob = require('fast-glob')` works, makes both VS Code and `tsc` happy. Thoughts @mweichert?